gui/vm/migrations/0005_auto_20171025_0450.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2017-10-25 04:50
+from __future__ import unicode_literals
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('vm', '0004_auto_20170525_0220'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='vm',
+            name='vm_type',
+            field=models.CharField(choices=[('Bhyve', 'Virtual Machine'), ('Container Provider', 'Docker VM')], default='Bhyve', max_length=50, verbose_name='VM Type'),
+        ),
+        migrations.AlterField(
+            model_name='vm',
+            name='memory',
+            field=models.IntegerField(help_text='Megabytes of RAM for the virtual machine.This memory will be allocated when the VM is running and not available to the host system or other VMs.', validators=[django.core.validators.MinValueValidator(512)], verbose_name='Memory Size (MiB)'),
```

Comment:
Space between the sentences and present tense:

'Megabytes of RAM for the virtual machine. This memory is allocated while the VM is running, and is not available to the host system or other VMs.'